packages/i18nify-js/src/modules/phoneNumber/getDialCodes.ts

```diff
@@ -0,0 +1,20 @@
+import { withErrorBoundary } from '../../common/errorBoundary';
+import { DIAL_CODE_MAPPER } from './data/dialCodeMapper';
+
+/**
+ * Retrieves a mapping of country codes to their respective international dial codes.
+ * @returns {Object} An object where each key is a country code (e.g., 'US', 'CA') and its value is the corresponding dial code (e.g., '+1' for 'US' and 'CA').
+ */
+const getDialCodes = (): { [key: string]: string } => {
+  const countryDialCode: { [key: string]: string } = {};
+
+  for (const [dialCode, countryCodes] of Object.entries(DIAL_CODE_MAPPER)) {
```

Comment:
change: why don't we just create a normal map and store it, instead of creating the dynamic map on the fly.

Reply:
Having a static map will contribute to bundle size. 
We need to make use of already available configs.